Week-5/Exercise5.4/main.test.js

```diff
@@ -0,0 +1,20 @@
+function add(a,b){
+    return (a+b);
+}
+test("adding two numbers", () => {
+    expect(add(2,2)).toBe(2+2)
```

Comment:
toBe(4)

think of more edge case, add(-1,0) toBe(-1)
add(-1.90932434,0) toBe(-1.90932434)